content/2021-online/register.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Register
+---
+
+## Register to attend
+
+As PromCon is colocated with KubeCon, the registration for PromCon Online 2021 is an add-on to the KubeCon ticket.
+
+
+<a class="btn btn-lg btn-default" href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/register/" target="_blank" role="button">
```

Comment:
You can also use this link directly: https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/register/